File: Api/Api_B84211_B87107/Entities/Citas.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Api_B84211_B87107.Entities
{
    public class Citas
    {
        [Key]
        public int id { get; set; }
        public int cedula { get; set; }
        public string centro_salud { get; set; }
        public string fecha { get; set; }
        public string especialidad { get; set; }
        public string descripción { get; set; }
        public string codigo { get; set; }

    }
}

File: Android/app/src/main/java/ucr/ac/cr/proyecto3_b87107_b84211/modelos/Citas.java
package ucr.ac.cr.proyecto3_b87107_b84211.modelos;

public class Citas {
    private int id;
    private int cedula;
    private String centro_salud;
    private String fecha;
    private String especialidad;
    private String descripción;
    private String codigo ;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getCedula() {
        return cedula;
    }

    public void setCedula(int cedula) {
        this.cedula = cedula;
    }

    public String getCentro_salud() {
        return centro_salud;
    }

    public void setCentro_salud(String centro_salud) {
        this.centro_salud = centro_salud;
    }

    public String getFecha() {
        return fecha;
    }

    public void setFecha(String fecha) {
        this.fecha = fecha;
    }

    public String getEspecialidad() {
        return especialidad;
    }

    public void setEspecialidad(String especialidad) {
        this.especialidad = especialidad;
    }

    public String getDescripción() {
        return descripción;
    }

    public void setDescripción(String descripción) {
        this.descripción = descripción;
    }

    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }
}
